helpers/dateTime.js

```diff
@@ -161,6 +161,44 @@ export function getUTCDateTimeFromLocalDateTime(date, time) {
 	return formatDateTimeInISO(utcDateTime);
 }
 
+export function getUTCDateTimeRange(type, diff) {
+	if (!type || (!diff && diff !== 0)) return {};
+	if (type !== 'hours' && type !== 'days' && type !== 'weeks' && type !== 'months') return {};
+
+	if (type === 'days' && diff === 0) {
+		// assume "today" (midnight in user's time to 23:59:59 in user's time)
+		const today = formatDateInISO(getToday());
+		const startValue = getUTCDateTimeFromLocalDateTime(today, '0:0:0');
+		const endValue = getUTCDateTimeFromLocalDateTime(today, '23:59:59');
+		return { startValue, endValue };
```

Comment:
This uses `getToday` which gets the local date based on the user's LMS timezone, then gets the UTC value of that date and midnight for start value and at 23:59:59 for end value. Confirmed with Jeff that these are the values we want even though this is the only preset that would involve future times.